lib/packwerk/file_processor.rb

```diff
@@ -1,6 +1,7 @@
 # typed: strict
 # frozen_string_literal: true
 
+require "parser/ast/node"
```

Comment:
I don't see usage of `AST::Node`. Can we just replace the other require in this file with this one?

Reply:
Makes sense, done!